controlplane/kubeadm/controllers/remediation.go

```diff
@@ -241,14 +235,17 @@ func (r *KubeadmControlPlaneReconciler) canSafelyRemoveEtcdMember(ctx context.Co
 		healthyMembers = append(healthyMembers, fmt.Sprintf("%s (%s)", etcdMember, machine.Name))
 	}
 
+	targetQuorum := targetTotalMembers/2.0 + 1
```

Comment:
```suggestion
	targetQuorum := (targetTotalMembers/2.0) + 1
```

I know it's the same, for readability, also please add a comment on why we're using this formula?